src/tests/ftest/osa/offline_extend.py

```diff
@@ -29,7 +30,8 @@ def setUp(self):
         self.test_oclass = None
         self.dmg_command.exit_status_exception = True
 
-    def run_offline_extend_test(self, num_pool, data=False, oclass=None):
+    def run_offline_extend_test(self, num_pool, data=False, oclass=None,
+                                exclude_or_drain=None):
```

Comment:
You might want to add description for `exclude_or_drain` argument.

Reply:
done